packages/node/src/transports/base.ts

```diff
@@ -55,17 +57,15 @@ export abstract class BaseTransport implements Transport {
   }
 
   /** Returns a build request option object used by request */
-  protected _getRequestOptions(address: string | url.URL): http.RequestOptions | https.RequestOptions {
-    if (!isString(address) || !(address instanceof url.URL)) {
-      throw new SentryError(`Incorrect transport url: ${address}`);
-    }
-
+  protected _getRequestOptions(url: url.URL): http.RequestOptions | https.RequestOptions {
     const headers = {
-      ...this._api.getRequestHeaders(SDK_NAME, SDK_VERSION),
+      // The auth headers are not included because auth is done via query string to match @sentry/browser.
+      //...this._api.getRequestHeaders(SDK_NAME, SDK_VERSION)
```

Comment:
This is a proposition. Without it, `this._api.getRequestHeaders` needs to know that the `Content-Type` header is not always `application/json` anymore, but depends on the event type (error != transaction).